javascript/src/seedsource/components/Layers.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+
+class Layers extends React.Component {
+    render() {
+        let { onToggleLayer, layers } = this.props
+        let layerList = (urlSegment) => layers.filter(layer => layer.urlTemplate.includes(urlSegment))
+                .map(layer =>   {
+                                    return <p key={layer.name}>
+                                                <input
+                                                    type="checkbox"
+                                                    value={layer.name}
+                                                    onChange={() => onToggleLayer(layer.name)}
+                                                    checked={layer.displayed}
+                                                    disabled={(layer.urlTemplate === "seedZone") ? true : false}
+                                                />
+                                                {layer.name}
+                                            </p>
+                                })
+        let [resultsLayer, seedZoneLayers, variableLayers] = ["{serviceId}", "seedZone", "{region}_{modelTime}"]
+            .map(layer => layerList(layer))
+
+        return (
+            <div className={"layers-tab"}>
+                {layers.length ? <p>Use this section to toggle layers on the map</p> :
+                    <p>Configure the map in the Tool tab before visiting this section</p>}
```

Comment:
This message shouldn't ever be shown, as we'll have a number of layers that are always available. In the event that there aren't, let's say "No layers available".